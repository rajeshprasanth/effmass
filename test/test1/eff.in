&CONTROL
  calculation='scf'
  outdir='./output',
  pseudo_dir='./'
  verbosity='high',
  tstress=.false.,
  tprnfor=.false.,
/

&SYSTEM
  ibrav=2,
  celldm(1)=10.2623466921d0,
  nat=2,
  ntyp=1,
  nbnd=16,
  ecutwfc=30.0d0,
  ecutrho=120.0d0
/

&ELECTRONS
  diagonalization='david',
  conv_thr=1d-06,
  mixing_mode='plain',
  mixing_beta=0.700d0,
/
&IONS
/
&CELL
/

ATOMIC_SPECIES
  Si 28.085500d0 Si.pz-vbc.UPF

ATOMIC_POSITIONS {alat}
  Si   0.0000000000d0   0.0000000000d0   0.0000000000d0
  Si   0.2500000000d0   0.2500000000d0   0.2500000000d0

K_POINTS {crystal}
8
0.5 0.5 0.5 1
0.5 0.5 0.0 1
0.5 0.0 0.5 1
0.5 0.0 0.0 1
0.0 0.5 0.5 1
0.0 0.5 0.0 1
0.0 0.0 0.5 1
0.0 0.0 0.0 1
