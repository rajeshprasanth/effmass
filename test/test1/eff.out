
     Program PWSCF v.6.3MaX starts on 19Jan2019 at  3:39:19 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Serial version
     Waiting for input...
     Reading input from standard input
Warning: card &IONS ignored
Warning: card / ignored
Warning: card &CELL ignored
Warning: card / ignored

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3

     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         433     433    127                 5985     5985     965



     bravais-lattice index     =            2
     lattice parameter (alat)  =      10.2623  a.u.
     unit-cell volume          =     270.1967 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     number of electrons       =         8.00
     number of Kohn-Sham states=           16
     kinetic-energy cutoff     =      30.0000  Ry
     charge density cutoff     =     120.0000  Ry
     convergence threshold     =      1.0E-06
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      =  SLA  PZ   NOGX NOGC ( 1  1  0  0 0 0)

     celldm(1)=  10.262347  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (  -0.500000   0.000000   0.500000 )  
               a(2) = (   0.000000   0.500000   0.500000 )  
               a(3) = (  -0.500000   0.500000   0.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = ( -1.000000 -1.000000  1.000000 )  
               b(2) = (  1.000000  1.000000  1.000000 )  
               b(3) = ( -1.000000  1.000000 -1.000000 )  


     PseudoPot. # 1 for Si read from file:
     ./Si.pz-vbc.UPF
     MD5 check sum: 6dfa03ddd5817404712e03e4d12deb78
     Pseudo is Norm-conserving, Zval =  4.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  431 points,  2 beta functions with: 
                l(1) =   0
                l(2) =   1

     atomic species   valence    mass     pseudopotential
        Si             4.00    28.08550     Si( 1.00)

     24 Sym. Ops. (no inversion) found
          (note: 24 additional sym.ops. were found but ignored
           their fractional translations are incommensurate with FFT grid)

                                    s                        frac. trans.

      isym =  1     identity                                     

 cryst.   s( 1) = (     1          0          0      )
                  (     0          1          0      )
                  (     0          0          1      )

 cart.    s( 1) = (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


      isym =  2     180 deg rotation - cart. axis [0,0,1]        

 cryst.   s( 2) = (     0          1         -1      )
                  (     1          0         -1      )
                  (     0          0         -1      )

 cart.    s( 2) = ( -1.0000000  0.0000000  0.0000000 )
                  (  0.0000000 -1.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


      isym =  3     180 deg rotation - cart. axis [0,1,0]        

 cryst.   s( 3) = (    -1          0          0      )
                  (    -1          0          1      )
                  (    -1          1          0      )

 cart.    s( 3) = ( -1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )


      isym =  4     180 deg rotation - cart. axis [1,0,0]        

 cryst.   s( 4) = (     0         -1          1      )
                  (     0         -1          0      )
                  (     1         -1          0      )

 cart.    s( 4) = (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000 -1.0000000  0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )


      isym =  5     120 deg rotation - cart. axis [-1,-1,-1]     

 cryst.   s( 5) = (    -1          0          1      )
                  (    -1          1          0      )
                  (    -1          0          0      )

 cart.    s( 5) = (  0.0000000  1.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )
                  (  1.0000000  0.0000000  0.0000000 )


      isym =  6     120 deg rotation - cart. axis [-1,1,1]       

 cryst.   s( 6) = (     0          1          0      )
                  (     0          0          1      )
                  (     1          0          0      )

 cart.    s( 6) = (  0.0000000 -1.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )
                  ( -1.0000000  0.0000000  0.0000000 )


      isym =  7     120 deg rotation - cart. axis [1,1,-1]       

 cryst.   s( 7) = (     1          0         -1      )
                  (     0          0         -1      )
                  (     0          1         -1      )

 cart.    s( 7) = (  0.0000000  1.0000000  0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )
                  ( -1.0000000  0.0000000  0.0000000 )


      isym =  8     120 deg rotation - cart. axis [1,-1,1]       

 cryst.   s( 8) = (     0         -1          0      )
                  (     1         -1          0      )
                  (     0         -1          1      )

 cart.    s( 8) = (  0.0000000 -1.0000000  0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )
                  (  1.0000000  0.0000000  0.0000000 )


      isym =  9     120 deg rotation - cart. axis [1,1,1]        

 cryst.   s( 9) = (     0          0         -1      )
                  (     0          1         -1      )
                  (     1          0         -1      )

 cart.    s( 9) = (  0.0000000  0.0000000  1.0000000 )
                  (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )


      isym = 10     120 deg rotation - cart. axis [-1,1,-1]      

 cryst.   s(10) = (    -1          1          0      )
                  (    -1          0          0      )
                  (    -1          0          1      )

 cart.    s(10) = (  0.0000000  0.0000000  1.0000000 )
                  ( -1.0000000  0.0000000  0.0000000 )
                  (  0.0000000 -1.0000000  0.0000000 )


      isym = 11     120 deg rotation - cart. axis [1,-1,-1]      

 cryst.   s(11) = (     0          0          1      )
                  (     1          0          0      )
                  (     0          1          0      )

 cart.    s(11) = (  0.0000000  0.0000000 -1.0000000 )
                  ( -1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )


      isym = 12     120 deg rotation - cart. axis [-1,-1,1]      

 cryst.   s(12) = (     1         -1          0      )
                  (     0         -1          1      )
                  (     0         -1          0      )

 cart.    s(12) = (  0.0000000  0.0000000 -1.0000000 )
                  (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000 -1.0000000  0.0000000 )


      isym = 13     inv. 180 deg rotation - cart. axis [1,1,0]   

 cryst.   s(13) = (     0          1          0      )
                  (     1          0          0      )
                  (     0          0          1      )

 cart.    s(13) = (  0.0000000 -1.0000000  0.0000000 )
                  ( -1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


      isym = 14     inv. 180 deg rotation - cart. axis [1,-1,0]  

 cryst.   s(14) = (     1          0         -1      )
                  (     0          1         -1      )
                  (     0          0         -1      )

 cart.    s(14) = (  0.0000000  1.0000000  0.0000000 )
                  (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


      isym = 15     inv.  90 deg rotation - cart. axis [0,0,-1]  

 cryst.   s(15) = (     0         -1          0      )
                  (     0         -1          1      )
                  (     1         -1          0      )

 cart.    s(15) = (  0.0000000 -1.0000000  0.0000000 )
                  (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )


      isym = 16     inv.  90 deg rotation - cart. axis [0,0,1]   

 cryst.   s(16) = (    -1          0          1      )
                  (    -1          0          0      )
                  (    -1          1          0      )

 cart.    s(16) = (  0.0000000  1.0000000  0.0000000 )
                  ( -1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )


      isym = 17     inv. 180 deg rotation - cart. axis [1,0,1]   

 cryst.   s(17) = (     1          0          0      )
                  (     0          0          1      )
                  (     0          1          0      )

 cart.    s(17) = (  0.0000000  0.0000000 -1.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )
                  ( -1.0000000  0.0000000  0.0000000 )


      isym = 18     inv. 180 deg rotation - cart. axis [-1,0,1]  

 cryst.   s(18) = (    -1          0          0      )
                  (    -1          1          0      )
                  (    -1          0          1      )

 cart.    s(18) = (  0.0000000  0.0000000  1.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )
                  (  1.0000000  0.0000000  0.0000000 )


      isym = 19     inv.  90 deg rotation - cart. axis [0,1,0]   

 cryst.   s(19) = (     0         -1          1      )
                  (     1         -1          0      )
                  (     0         -1          0      )

 cart.    s(19) = (  0.0000000  0.0000000 -1.0000000 )
                  (  0.0000000 -1.0000000  0.0000000 )
                  (  1.0000000  0.0000000  0.0000000 )


      isym = 20     inv.  90 deg rotation - cart. axis [0,-1,0]  

 cryst.   s(20) = (     0          1         -1      )
                  (     0          0         -1      )
                  (     1          0         -1      )

 cart.    s(20) = (  0.0000000  0.0000000  1.0000000 )
                  (  0.0000000 -1.0000000  0.0000000 )
                  ( -1.0000000  0.0000000  0.0000000 )


      isym = 21     inv. 180 deg rotation - cart. axis [0,1,1]   

 cryst.   s(21) = (     1         -1          0      )
                  (     0         -1          0      )
                  (     0         -1          1      )

 cart.    s(21) = (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )
                  (  0.0000000 -1.0000000  0.0000000 )


      isym = 22     inv. 180 deg rotation - cart. axis [0,1,-1]  

 cryst.   s(22) = (     0          0          1      )
                  (     0          1          0      )
                  (     1          0          0      )

 cart.    s(22) = (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )


      isym = 23     inv.  90 deg rotation - cart. axis [-1,0,0]  

 cryst.   s(23) = (     0          0         -1      )
                  (     1          0         -1      )
                  (     0          1         -1      )

 cart.    s(23) = ( -1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )


      isym = 24     inv.  90 deg rotation - cart. axis [1,0,0]   

 cryst.   s(24) = (    -1          1          0      )
                  (    -1          0          1      )
                  (    -1          0          0      )

 cart.    s(24) = ( -1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )
                  (  0.0000000 -1.0000000  0.0000000 )


     point group T_d (-43m) 
     there are  5 classes
     the character table:

       E     8C3   3C2   6S4   6s_d 
A_1    1.00  1.00  1.00  1.00  1.00
A_2    1.00  1.00  1.00 -1.00 -1.00
E      2.00 -1.00  2.00  0.00  0.00
T_1    3.00  0.00 -1.00  1.00 -1.00
T_2    3.00  0.00 -1.00 -1.00  1.00

     the symmetry operations in each class and the name of the first element:

     E        1
          identity                                               
     8C3      5    7    8    6   12    9   10   11
          120 deg rotation - cart. axis [-1,-1,-1]               
     3C2      2    3    4
          180 deg rotation - cart. axis [0,0,1]                  
     6S4     15   16   20   19   23   24
          inv.  90 deg rotation - cart. axis [0,0,-1]            
     6s_d    13   14   17   18   21   22
          inv. 180 deg rotation - cart. axis [1,1,0]             

   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Si  tau(   1) = (   0.0000000   0.0000000   0.0000000  )
         2           Si  tau(   2) = (   0.2500000   0.2500000   0.2500000  )

   Crystallographic axes

     site n.     atom                  positions (cryst. coord.)
         1           Si  tau(   1) = (  0.0000000  0.0000000  0.0000000  )
         2           Si  tau(   2) = ( -0.2500000  0.7500000 -0.2500000  )

     number of k points=     8
                       cart. coord. in units 2pi/alat
        k(    1) = (  -0.5000000   0.5000000   0.5000000), wk =   0.2500000
        k(    2) = (   0.0000000   0.0000000   1.0000000), wk =   0.2500000
        k(    3) = (  -1.0000000   0.0000000   0.0000000), wk =   0.2500000
        k(    4) = (  -0.5000000  -0.5000000   0.5000000), wk =   0.2500000
        k(    5) = (   0.0000000   1.0000000   0.0000000), wk =   0.2500000
        k(    6) = (   0.5000000   0.5000000   0.5000000), wk =   0.2500000
        k(    7) = (  -0.5000000   0.5000000  -0.5000000), wk =   0.2500000
        k(    8) = (   0.0000000   0.0000000   0.0000000), wk =   0.2500000

                       cryst. coord.
        k(    1) = (   0.5000000   0.5000000   0.5000000), wk =   0.2500000
        k(    2) = (   0.5000000   0.5000000   0.0000000), wk =   0.2500000
        k(    3) = (   0.5000000   0.0000000   0.5000000), wk =   0.2500000
        k(    4) = (   0.5000000   0.0000000   0.0000000), wk =   0.2500000
        k(    5) = (   0.0000000   0.5000000   0.5000000), wk =   0.2500000
        k(    6) = (   0.0000000   0.5000000   0.0000000), wk =   0.2500000
        k(    7) = (   0.0000000   0.0000000   0.5000000), wk =   0.2500000
        k(    8) = (   0.0000000   0.0000000   0.0000000), wk =   0.2500000

     Dense  grid:     5985 G-vectors     FFT dimensions: (  25,  25,  25)

     Dynamical RAM for                 wfc:       0.18 MB

     Dynamical RAM for     wfc (w. buffer):       1.65 MB

     Dynamical RAM for           str. fact:       0.09 MB

     Dynamical RAM for           local pot:       0.00 MB

     Dynamical RAM for          nlocal pot:       0.09 MB

     Dynamical RAM for                qrad:       0.12 MB

     Dynamical RAM for          rho,v,vnew:       0.63 MB

     Dynamical RAM for               rhoin:       0.21 MB

     Dynamical RAM for            rho*nmix:       1.46 MB

     Dynamical RAM for           G-vectors:       0.39 MB

     Dynamical RAM for          h,s,v(r/c):       0.19 MB

     Dynamical RAM for          <psi|beta>:       0.00 MB

     Dynamical RAM for                 psi:       0.73 MB

     Dynamical RAM for                hpsi:       0.73 MB

     Dynamical RAM for      wfcinit/wfcrot:       0.37 MB

     Estimated static dynamical RAM per process >       5.42 MB

     Estimated max dynamical RAM per process >       7.08 MB

     Initial potential from superposition of free atoms

     starting charge    7.99901, renormalised to    8.00000
     Starting wfcs are    8 randomized atomic wfcs +    8 random wfcs

     total cpu time spent up to now is        0.1 secs

     Self-consistent Calculation

     iteration #  1     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  5.8

     total cpu time spent up to now is        0.5 secs

     total energy              =     -15.65524875 Ry
     Harris-Foulkes estimate   =     -15.69781814 Ry
     estimated scf accuracy    <       0.11627858 Ry

     iteration #  2     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.45E-03,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.7 secs

     total energy              =     -15.66239141 Ry
     Harris-Foulkes estimate   =     -15.66276342 Ry
     estimated scf accuracy    <       0.00319512 Ry

     iteration #  3     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.99E-05,  avg # of iterations =  3.4

     total cpu time spent up to now is        0.8 secs

     total energy              =     -15.66249583 Ry
     Harris-Foulkes estimate   =     -15.66249667 Ry
     estimated scf accuracy    <       0.00009393 Ry

     iteration #  4     ecut=    30.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.17E-06,  avg # of iterations =  3.8

     total cpu time spent up to now is        1.0 secs

     End of self-consistent calculation

          k =-0.5000 0.5000 0.5000 (   754 PWs)   bands (ev):

    -3.2926  -0.7420   5.1062   5.1062   7.7423   9.5085   9.5085  13.6377
    16.7492  16.7492  17.3844  17.4985  17.4985  19.0038  25.1474  25.9140

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   0.0000   0.0000   0.0000   0.0000
     0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000

          k = 0.0000 0.0000 1.0000 (   740 PWs)   bands (ev):

    -1.4907  -1.4907   3.4023   3.4023   6.8028   6.8028  16.2679  16.2679
    17.1837  17.1837  18.7349  18.7350  19.0743  19.0743  24.7603  24.7603

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   0.0000   0.0000   0.0000   0.0000
     0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000

          k =-1.0000 0.0000 0.0000 (   740 PWs)   bands (ev):

    -1.4907  -1.4907   3.4023   3.4023   6.8028   6.8028  16.2679  16.2679
    17.1837  17.1837  18.7349  18.7349  19.0742  19.0743  24.7604  24.7604

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   0.0000   0.0000   0.0000   0.0000
     0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000

          k =-0.5000-0.5000 0.5000 (   754 PWs)   bands (ev):

    -3.2926  -0.7420   5.1062   5.1062   7.7423   9.5085   9.5085  13.6377
    16.7492  16.7492  17.3844  17.4985  17.4985  19.0039  25.1474  25.9140

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   0.0000   0.0000   0.0000   0.0000
     0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000

          k = 0.0000 1.0000 0.0000 (   740 PWs)   bands (ev):

    -1.4907  -1.4907   3.4023   3.4023   6.8028   6.8028  16.2679  16.2679
    17.1837  17.1837  18.7349  18.7349  19.0742  19.0743  24.7603  24.7603

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   0.0000   0.0000   0.0000   0.0000
     0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000

          k = 0.5000 0.5000 0.5000 (   754 PWs)   bands (ev):

    -3.2926  -0.7420   5.1062   5.1062   7.7423   9.5085   9.5085  13.6377
    16.7492  16.7492  17.3844  17.4985  17.4985  19.0038  25.1474  25.9140

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   0.0000   0.0000   0.0000   0.0000
     0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000

          k =-0.5000 0.5000-0.5000 (   754 PWs)   bands (ev):

    -3.2926  -0.7420   5.1062   5.1062   7.7423   9.5085   9.5085  13.6377
    16.7492  16.7492  17.3844  17.4985  17.4985  19.0038  25.1474  25.9140

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   0.0000   0.0000   0.0000   0.0000
     0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000

          k = 0.0000 0.0000 0.0000 (   749 PWs)   bands (ev):

    -5.6719   6.3339   6.3339   6.3339   8.7627   8.7627   8.7627   9.6081
    13.8253  13.9570  13.9570  17.2411  17.2411  17.2411  21.0943  28.9642

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   0.0000   0.0000   0.0000   0.0000
     0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000   0.0000

     highest occupied, lowest unoccupied level (ev):     6.3339    6.8028

!    total energy              =     -15.66249996 Ry
     Harris-Foulkes estimate   =     -15.66249991 Ry
     estimated scf accuracy    <       0.00000005 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =       4.74398797 Ry
     hartree contribution      =       1.25425461 Ry
     xc contribution           =      -4.86365480 Ry
     ewald contribution        =     -16.79708774 Ry

     convergence has been achieved in   4 iterations

     Writing output data file pwscf.save/

     init_run     :      0.10s CPU      0.10s WALL (       1 calls)
     electrons    :      0.87s CPU      0.88s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.06s CPU      0.07s WALL (       1 calls)
     wfcinit:atom :      0.00s CPU      0.00s WALL (       8 calls)
     wfcinit:wfcr :      0.06s CPU      0.06s WALL (       8 calls)
     potinit      :      0.00s CPU      0.00s WALL (       1 calls)
     hinit0       :      0.03s CPU      0.03s WALL (       1 calls)

     Called by electrons:
     c_bands      :      0.76s CPU      0.77s WALL (       4 calls)
     sum_band     :      0.10s CPU      0.10s WALL (       4 calls)
     v_of_rho     :      0.01s CPU      0.01s WALL (       5 calls)
     v_h          :      0.00s CPU      0.00s WALL (       5 calls)
     v_xc         :      0.01s CPU      0.01s WALL (       5 calls)
     mix_rho      :      0.00s CPU      0.00s WALL (       4 calls)

     Called by c_bands:
     init_us_2    :      0.01s CPU      0.01s WALL (      72 calls)
     cegterg      :      0.76s CPU      0.76s WALL (      32 calls)

     Called by sum_band:

     Called by *egterg:
     h_psi        :      0.58s CPU      0.59s WALL (     151 calls)
     g_psi        :      0.00s CPU      0.00s WALL (     111 calls)
     cdiaghg      :      0.05s CPU      0.05s WALL (     143 calls)
     cegterg:over :      0.07s CPU      0.07s WALL (     111 calls)
     cegterg:upda :      0.05s CPU      0.05s WALL (     111 calls)
     cegterg:last :      0.03s CPU      0.03s WALL (      39 calls)

     Called by h_psi:
     h_psi:pot    :      0.58s CPU      0.59s WALL (     151 calls)
     h_psi:calbec :      0.01s CPU      0.01s WALL (     151 calls)
     vloc_psi     :      0.55s CPU      0.56s WALL (     151 calls)
     add_vuspsi   :      0.01s CPU      0.01s WALL (     151 calls)

     General routines
     calbec       :      0.01s CPU      0.01s WALL (     151 calls)
     fft          :      0.00s CPU      0.00s WALL (      15 calls)
     ffts         :      0.00s CPU      0.00s WALL (       4 calls)
     fftw         :      0.59s CPU      0.60s WALL (    3620 calls)


     PWSCF        :     1.01s CPU         1.05s WALL


   This run was terminated on:   3:39:20  19Jan2019            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
